session1/examples/window-example.js

```diff
@@ -0,0 +1,5 @@
+function getName(name) {
```

Comment:
Why are we using ES5 function syntax instead of ES6 arrow function?

Reply:
Modified